modules/rosa-sts-account-wide-roles-and-policies.adoc

```diff
@@ -730,6 +730,44 @@ The account number present in the `sts_installer_trust_policy.json` and `sts_sup
 ----
 ====
 
+[discrete]
+[id="rosa-sts-account-wide-roles-and-policies-example-cli-output-for-policies-attached-to-a-role_{context}"]
+==== Example CLI output for policies attached to a role 
+
+When a policy is attached to a role, the ROSA CLI displays confirmation output. The output depends on the type of policy.
+
+* If a policy is a trust policy, it outputs the content of the policy.
```

Comment:
```suggestion
* If the policy is a trust policy, the ROSA CLI outputs the content of the policy.
```

Reply:
should be:
* If the policy is a trust policy, the ROSA CLI outputs the role name and the content of the policy.